common/commands.h

```diff
@@ -111,4 +111,100 @@ namespace commands::assets {
 
 // =====================================================================================================================
 
+namespace disco::commands::config {
+    static constexpr const char* Subject = "config";
+
+    class In : public pack::Node
+    {
+    public:
+        pack::String     type          = FIELD("FTY_DISCOVERY_TYPE");
+        pack::StringList scans         = FIELD("FTY_DISCOVERY_SCANS");
+        pack::StringList ips           = FIELD("FTY_DISCOVERY_IPS");
+        pack::StringList docs          = FIELD("FTY_DISCOVERY_DOCUMENTS");
+        pack::String     status        = FIELD("FTY_DISCOVERY_DEFAULT_VALUES_STATUS");
+        pack::Int32      priority      = FIELD("FTY_DISCOVERY_DEFAULT_VALUES_PRIORITY");
+        pack::String     parent        = FIELD("FTY_DISCOVERY_DEFAULT_VALUES_PARENT");
+        pack::String     linkSrc       = FIELD("FTY_DISCOVERY_DEFAULT_VALUES_LINK_SRC");
+        pack::StringList scansDisabled = FIELD("FTY_DISCOVERY_SCANS_DISABLED");
+        pack::StringList ipsDisabled   = FIELD("FTY_DISCOVERY_IPS_DISABLED");
+        pack::StringList protocols     = FIELD("FTY_DISCOVERY_PROTOCOLS");
+        pack::Int32      dumpPool      = FIELD("FTY_DISCOVERY_DUMP_POOL");
+        pack::Int32      scanPool      = FIELD("FTY_DISCOVERY_SCAN_POOL");
+        pack::Int32      scanTimeout   = FIELD("FTY_DISCOVERY_SCAN_TIMEOUT");
+        pack::Int32      dumpLooptime  = FIELD("FTY_DISCOVERY_DUMP_LOOPTIME");
+
+    public:
+        using pack::Node::Node;
+        META(In, type, scans, ips, docs, status, priority, parent, linkSrc, scansDisabled, ipsDisabled, protocols,
+            dumpPool, scanPool, scanTimeout, dumpLooptime);
+    };
+
+} // namespace disco::commands::config
+
+// =====================================================================================================================
+
+namespace disco::commands::scan {
+    static constexpr const char* Subject = "scan";
+
+    enum class Status
+    {
+        CancelledByUser = 1,
+        Terminated      = 2,
+        InProgress      = 3,
+        Unknown
```

Comment:
Better 'unknown' as a zero... otherwise CancelledByUser will be default value in pack and omitted in serialization/deserialization.